### PR TITLE
brca-exchange & jax clinical trials harvesters

### PR DESCRIPTION
**Pull request by bwalsh**
This PR:
* contains two new harvesters:
  * brca - harvests brca-exchange's expert-reviewed [variants](http://brcaexchange.org/variants) #59 
  * jax_trials - harvests jax's  [clinical trials](https://ckb.jax.org/clinicalTrial/advancedFindInit) #57 

Discussion:
brca:
* mapping brca exchange's [clinical significance](http://brcaexchange.org/help#clinical-significance) to g2p's oncogenic
* determining g2p `evidence_label`?  These variant's have had 'expert review'.  Some have associated pubmed articles.  Leave evidence_lable null, or ... ?
* normalizing disease? brca oncogenic evidence is mapped to phenotypes OMIM:
  * BREAST-OVARIAN CANCER, FAMILIAL, SUSCEPTIBILITY TO, 1; BROVCA1 (604370)	Breast Cancer
  * BREAST-OVARIAN CANCER, FAMILIAL, SUSCEPTIBILITY TO, 2; BROVCA2 (612555)	Breast Cancer
  `for now` I have aliased them to 'breast cancer'.  Is this the best choice?

jax_trials:
* hardcoded evidence level 'D'

### CODE REVIEW

**Comment by grmayfie** [comment]
Looking at brca and jax_trials on my local, and it looks like currently brca description is mapped to `Pathogenicity_expert`, and I think it should be mapped to `Comment_on_clinical_significance_ENIGMA` which is a much more in depth description, with `Pathogenicity_expert` mapped to the `oncogenic` flag instead. It also looks like feature.name could use a different mapping. Perhaps`Genomic_Coordinate_hg37` instead of `Protein_Change`?  It also has more information. Unless there was a particular reason the other mappings were chosen? Thoughts?

Also, it looks like jax_trials was written based on `jax.py` without the additional recent variant/biomarker updates, which will likely benefit it, since it appears to have many of the same errors as jax. Working on that update.

**Comment by bwalsh** [comment]
@mayfielg thanks for the analysis.  your points on brca-exchange are well taken.  I'll look at it.
In the meantime, can you base your work on this combined branch?  thanks

**Comment by grmayfie** [comment]
Sure thing. I'll look at it again tomorrow.

**Comment by grmayfie** [comment]
@bwalsh I'm having trouble with the alterations made to the nginx setup. How is the aws instance versus the local instance being specified?

**Comment by bwalsh** [comment]
@mayfielg  `docker-compose -f docker-compose.yml -f cloud-setup/docker-compose-nginx.yml`
should bring up the nginx front end
